src/website/src/website-components/Icon/IconCard.jsx

```diff
@@ -1,4 +1,4 @@
-/* eslint-disable import/no-extraneous-dependencies */
+/* eslint-disable import/no-extraneous-dependencies, react/no-danger */
```

Comment:
Also a warning, no need to ignore the `react/no-danger`

Reply:
removed, it's my editor who complains about these.